packages/frontend/src/features/scheduler/hooks/useScheduler.ts

```diff
@@ -184,7 +189,7 @@ export const useSendNowScheduler = () => {
                     );
                 }
             },
-            onError: async (error: { error: Error }) => {
+            onError: async ({ error }) => {
```

Comment:
I think this should be: 
```suggestion
            onError: async (error) => {
```

Reply:
And this is the correct typing for this `useQuery`

```
const { data: scheduledDeliveryJobStatus } = useQuery<
        ApiJobStatusResponse['results'] | undefined,
        ApiErrorPayload,
        ApiJobStatusResponse['results']
    >(
```

Reply:
Fixed with type + use new method.